app/api/admin_translations.py

```diff
@@ -0,0 +1,21 @@
+from flask import send_file, Blueprint
+import os
+import shutil
+import uuid
+
+app_dir = os.path.dirname(os.path.abspath(os.path.dirname(__file__)))
+file_dir = os.path.join(app_dir, 'translations')
```

Comment:
Use os.tmpdir

Reply:
Please correct me if I'm wrong, I don't think there exists anything like os.tmpdir. What should I be doing here?